tests/ui/macros/macros-nonfatal-errors.stderr

```diff
@@ -103,8 +115,14 @@ help: try removing these
    |
 LL |     #[default]
    |     ^^^^^^^^^^
+help: try removing these
```

Comment:
Why is this rendering as several helps rather than one multispan?

Reply:
Updated the derives to allow `#[help]` on MultiSpan